docs/src/pages/components/autocomplete/Asynchronous.tsx

```diff
@@ -1,5 +1,4 @@
 // *https://www.registers.service.gov.uk/registers/country/use-the-api*
-import fetch from 'cross-fetch';
```

Comment:
Should we keep it for the demos? So it work when people copy & paste or leave it up to them? (no preference)

Reply:
It's not a good polyfill strategy anyway so I'm happy to remove it.